webapp/home/subdomains/genome/content/assembly.py

```diff
@@ -339,42 +339,9 @@
                 'label': '<code>GenomeScope</code> genome profile summary',
             },
         ],
-        "view_link": "https://dockstore.org/workflows/github.com/Delphine-L/iwc/WF4-Assembly_with_HiC:VGP",
+        "view_link": "https://dockstore.org/workflows/github.com/iwc-workflows/Assembly-Hifi-HiC-phasing-VGP4/main:main",
         "view_tip": view_workflow_tip,
-        "button_link": "https://genome.usegalaxy.org.au/workflows/trs_import?trs_server=dockstore.org&trs_id=%23workflow/github.com/Delphine-L/iwc/WF4-Assembly_with_HiC",
-        "button_tip": import_workflow_tip,
-    },
-    {
-        "title_html": 'Purge duplicates',
```

Comment:
Did you meant to remove this Purge duplicates workflow?

Reply:
Yes, thanks